platform/ios/Mapbox.playground/Contents.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+import XCPlayground
+import Mapbox
+
+//: # Mapbox Maps
+
+/*:
+### Setting the Access Token
+ 
+Add a PlaygroundInfo.plist file to the Resources folder in this playground. Then add an 
+access token key value pair:
```

Comment:
Here’s another approach that would have the same effect of placing an ignored file in the Resources folder. I’m not sure if it’s more or less convenient, but perhaps we could use this approach for other purposes in the future:

```swift
/*:
Put your access token into a plain text file called `token`. Then select the “token” placeholder below, go to Editor ‣ Insert File Literal, and select the `token` file.
*/
let accessToken = String(contentsOfURL: <#token#>)
```